src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
+$mainAccent:#060b35;
+$secondary: #616070;
+$blueAccent: #00acdc;
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__content {
+    margin: 32px 16px 16px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__name {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainAccent;
+    margin: 40px 0 0;
+  }
+
+  &__productCode {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+    margin: 4px 0 0;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 0;
+  }
+
+  &__stars {
+    display: flex;
+
+    &--4 :nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+  }
+
+  &__star:first-child {
+    margin-left: 0;
+  }
+
+  &__number {
+    margin: 0;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
```suggestion

```

Reply:
Will it be taken by default?

Reply:
yes, until you set default styles by yourself